src/tabs/motors.html

```diff
@@ -229,6 +229,15 @@
                                     <div class="helpicon cf_tip" i18n_title="govSpoolupTimeHelp"></div>
                                 </label>
                             </div>
+                            <div class="field govEnabled" id="govMinSpoolupThrottle">
+                                <label>
+                                    <div class="field_box">
+                                        <input type="number" id="govSpoolupMinThrottle" min="0" max="50" step="1" />
+                                    </div>
```

Comment:
`id="govMinSpoolupThrottle"` **must** only be used once. This usage is preventing the field from being hidden as intended. JQuery `.closest()` is useful for selecting the other element.